services/security-headers/security-headers.service.js

```diff
@@ -53,6 +63,7 @@ module.exports = class SecurityHeaders extends BaseService {
       D: 'orange',
       E: 'orange',
       F: 'red',
+      R: 'lightgrey',
```

Comment:
```suggestion
      R: 'blue',
```

:+1: for blue. Normally we use blue for an blue for an information status and lightgrey for invalid.